x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -71,6 +72,12 @@ export const SearchIndexDetailsPage = () => {
       await playgroundLocator.navigate({ 'default-index': index.name });
     }
   }, [share, index]);
+  const navigateToDiscover = useCallback(async () => {
```

Comment:
can this be extracted out into a hook so its re-used in the data component aswell

Reply:
Of course